dotnet/src/Connectors/Connectors.AzureOpenAI/Core/AzureClientCore.ChatCompletion.cs

```diff
@@ -49,6 +49,7 @@ protected override ChatCompletionOptions CreateChatCompletionOptions(
             EndUserId = executionSettings.User,
             TopLogProbabilityCount = executionSettings.TopLogprobs,
             IncludeLogProbabilities = executionSettings.Logprobs,
+            AllowParallelToolCalls = toolCallingConfig.Options?.AllowParallelCalls
```

Comment:
Should we do the following to avoid any future risk of sending a null value

```
if (toolCallingConfig.Options?.AllowParallelCalls is not null)
{
    options.AllowParallelToolCalls = toolCallingConfig.Options?.AllowParallelCalls;
}
```

Reply:
Yep, the property used to be init only; now it has setter.